xml/System.Threading.Tasks/ValueTask`1.xml

```diff
@@ -377,9 +392,17 @@
         <ReturnType>TResult</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the result.</summary>
+        <value>The result.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks  
+ If this <xref:System.Threading.Tasks.ValueTask%601> wraps a successful result, this property directly returns it.
+
+ If it wraps a <xref:System.Threading.Tasks.Task%601>, the behavior of <xref:System.Threading.Tasks.ValueTask%601.Result%2A> is similar to the behavior of accessing <xref:System.Threading.Tasks.Task%601.Result%2A> on the wrapped task: if the task hasn't completed, accessing the property blocks the calling thread until it completes; if the task has completed successfully, the property returns the result; if the task has completed unsuccessfully, accessing the property throws an exception. The thrown exception is not wrapped in an <xref:System.AggregateException>, which is different from the behavior of <xref:System.Threading.Tasks.Task%601.Result%2A> in the same situation.
```

Comment:
if the task has completed unsuccessfully --> if the task has faulted or was cancelled